app/api_2_0/models.py

```diff
@@ -49,39 +49,34 @@ def delete_user_table(self, connection):
             return {'Status': False, 'Message': e}
 
-    def create_db_connection(self):
-        """Connect to the db."""
-        connection = None
-        try:
-            connection = psycopg2.connect("dbname='rides' user='rider' \
-            password='pass123' host='localhost' port='5432'")
-        except Exception as e:
-            raise e
-        return connection
-
-    def create_user(self, connection, email, password, confirm_password, user_type):
+    def create_user(self, connection, user_data):
         """Create user record."""
-        if self.find_user(email).get('Status'):
+        email = user_data.get('email')
+        password = user_data.get('password')
+        confirm_password = user_data.get('confirm_password')
+        user_type = user_data.get('user_type')
+
+        if self.find_user(connection, email).get('Status'):
             return {'Status': False, 'Message': 'User already exists'}
         conn = connection
         cursor = conn.cursor()
         try:
-            if not is_empty(email):
-                if User.verify_email(email):
-                    if User.check_password_length(password):
-                        if User.confirm_password(password, confirm_password):
-                            user_pass = BCRYPT.generate_password_hash(password)
-                            query = "insert into users (email, user_password, driver) values ('{}', '{}', '{}');".format(email, password, user_type)
-                            cursor.execute(query)
-                            conn.commit()
-                            conn.close()
-                            return {"Status": True, "Message": "Succesfuly created user record"}
-                        return {"Status": False, "Message": "Passwords dont match!"}
-                    return {"Status": False, "Message": "Password should not be less than six characters"}
-                return {"Status": False, "Message": "Invalid email Address!"}
-            return {"Status": False, "Message": "Email cannot be blank!"}
+            if User.verify_email(email):
+                if User.check_password_length(password):
+                    if User.confirm_password(password, confirm_password):
+                        user_pass = BCRYPT.generate_password_hash(password)
+                        query = "INSERT INTO users (email, user_password, driver) \
+                                VALUES ('{}', '{}', '{}')".\
+                                format(email, password, user_type)
+                        cursor.execute(query)
+                        conn.commit()
+                        conn.close()
+                        return {"Status": True, "Message": "Succesfuly created user record"}
+                    return {"Status": False, "Message": "Passwords dont match!"}
+                return {"Status": False, "Message": "Password should not be less than six characters"}
+            return {"Status": False, "Message": "Invalid email Address!"}
```

Comment:
Just a suggestion: You could move the error checks into functions for readability. I think the nesting makes it hard to identify bugs.
What if a user inputs most of the fields with errors, wouldn't it be a good idea to bundle the errors so that the user can rectify all of them at once?